src/cloud-api-adaptor/docker/kind_cluster.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+
+# Ref: https://stackoverflow.com/questions/299728/how-do-you-use-newgrp-in-a-script-then-stay-in-that-group-when-the-script-exits
+newgrp docker <<EOF
+
+# Accept two arguments: create and delete
+# create: creates a kind cluster
+# delete: deletes a kind cluster
+
+CLUSTER_NAME="${CLUSTER_NAME:-kind}"
```

Comment:
Could we consider defaulting this to `peer-pods`, like the e2e properties does. Then if a use needs to clean up an e2e created cluster (like I just need as something went bad with the snapshotter), they can just do `./docker/kind_cluster.sh delete`, rather than needing to look up what the e2e calls it?